@rickystewart Flaky test issue?  How do I re-trigger CI? (Although it's marked optional.)

```
Failed
=== RUN   TestTenantRateLimiter
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/e6186277801b886603091d97fcff3917/logTestTenantRateLimiter944326363
    test_log_scope.go:80: use -show-logs to present logs inline
    client_tenant_test.go:248: 
        	Error Trace:	client_tenant_test.go:248
        	Error:      	
        	Test:       	TestTenantRateLimiter
    panic.go:613: -- test log scope end --
--- FAIL: TestTenantRateLimiter (16.02s)
```

--- a/AUTHORS
+++ b/AUTHORS
@@ -359,6 +359,7 @@ Solon Gordon <solon@cockroachlabs.com> solongordon <solongordon@gmail.com>
 Song Hao <songhao9021@gmail.com> songhao <songhao9021@gmail.com>
 Spas Bojanov <spas@cockroachlabs.com> <pachob@gmail.com>
 Spencer Kimball <spencer.kimball@gmail.com> <spencer@cockroachlabs.com>
+Stan Rosenberg <stan.rosenberg@gmail.com> <stan@cockroachlabs.com>
 Stephanie Bodoff <stephanie@cockroachlabs.com>
 Stephen Mooney <stephen.mooney@gmail.com> <stephen@cockroachlabs.com>
 Steven Danna <danna@cockroachlabs.com>

